core/assets/sounds/cues/builtToScaleFever/data.json

```diff
@@ -40,31 +40,31 @@
       "name": "8 bounces",
       "cues": [
         {
-          "id": "builtToScaleFever/1",
+          "id": "builtToScaleFever/4",
```

Comment:
~who wants to count the number of changes this has had over its lifetime~

the changes are fine